Mathlib/CategoryTheory/Monoidal/Cartesian/Mod_.lean

```diff
@@ -0,0 +1,28 @@
+/-
+Copyright (c) 2025 Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Lezeau
+-/
+import Mathlib.CategoryTheory.ChosenFiniteProducts
+import Mathlib.CategoryTheory.Monoidal.Mod_
+
+/-!
+# Additional results about module objects in cartesian monoidal categories
+-/
+
+open CategoryTheory MonoidalCategory ChosenFiniteProducts
+
+universe v u
+variable {C : Type u} [Category.{v} C] [ChosenFiniteProducts C]
+
+attribute [local simp] leftUnitor_hom
```

Comment:
@erdOne, since #24399, I need this lemma to be simp for the proofs below to work. Shouldn't it be simp globally?